cmd/dagger/cmd/common/common.go

```diff
@@ -82,6 +84,21 @@ func ValueDocOneLine(val *compiler.Value) string {
 	return strings.Join(docs, " ")
 }
 
+func convertRelativePaths(cacheOptionEntries []buildkit.CacheOptionsEntry) []buildkit.CacheOptionsEntry {
+	var convertedOptions = cacheOptionEntries
+	var pathableAttributes = []string{"src", "dest"}
+
+	for _, option := range convertedOptions {
+		for _, key := range pathableAttributes {
+			if _, ok := option.Attrs[key]; ok {
+				option.Attrs[key], _ = homedir.Expand(option.Attrs[key])
```

Comment:
It's better not to ignore the error.

If you want to avoid returning an error to keep this function simple, I would add a `if err != nil { panic(err) }`